frontend/packages/knative-plugin/integration-tests/features/managed-kafka connection/managed-services-kafka-connection.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Support for Managed Services
+              As a user, I want to create the Kafka connection and support the manage services
+
+        Background:
+            Given user is at developer perspective
+              And user has installed RHOAS operator
+              And user has created or selected namespace "aut-managed-services"
+
+
+        @regression, @manual
+        Scenario: Create the Managed Services from Add page
+            Given user is at Add page
+             When user clicks Managed Services card
+              And user clicks "Red Hat Cloud Services" card
+              And user clicks on link "https://cloud.redhat.com/openshift/token"
+              And user copies the API token from the link
+              And user returns to console
+              And user pastes the token in the API Token
+              And user clicks on Connect button
+              And user clicks on close buton
+              And user clicks "Red Hat Cloud Services" card again
+              And user clicks on Connect button
+              And user selects the Kafka Instance
+              And user clicks on Next button
+             Then user will be redirected to Topology page
+              And user is able to see Kafka Connection in topology page
```

Comment:
would you add here about how they can click on the topology item and see information in the drawer, and also link to the kafka in the data plane ?

Reply:
Are you talking about checking the sidebar of the Kafka connection instance here?

Reply:
yes